server/models/SentenceAndOffenceViewModel.ts

```diff
@@ -72,4 +76,12 @@ export default class SentenceAndOffenceViewModel {
   public isErsedChecked(): boolean {
     return this.userInputs?.calculateErsed === true
   }
+
+  public isErsedElligible(): boolean {
```

Comment:
Minor tip - kotlin  has a shorthand way of writing these one line return methods; e.g. 
`public isErsedElligible(): boolean = this.sentencesAndOffences.filter(sentence => SentenceTypes.isSentenceErsedElligible(sentence)).length > 0`